k6LoadTests.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable */
+import { sleep } from 'k6';
+import http from 'k6/http';
+
+export const options = {
+  duration: '10m',
+  vus: 20,
+  thresholds: {
+    //avg is around 800ms on https://tapahtumat.test.kuva.hel.ninja
+    http_req_duration: ['p(95)<5000'],
```

Comment:
5 seconds seems to be way too long for a normal interface interaction. As the response time distribution shape is unknown we cannot presume that p(95)<5000 guarantees an average of 800ms. My suggestion would be to look at the median,

Reply:
currently i think the first version of the test is not even for performance, but for availability in general. We dont have a clear definition of failure atm. Any suggestions are welcome though